code/nebula/neb.h

```diff
@@ -41,16 +41,15 @@ extern float Neb2_awacs;
 extern float Neb2_fog_near_mult;
 extern float Neb2_fog_far_mult;
 
-#define NEB_FOG_VISIBILITY_MULT_TRAIL			1.0f
-#define NEB_FOG_VISIBILITY_MULT_THRUSTER		1.5f
-#define NEB_FOG_VISIBILITY_MULT_WEAPON			1.3f
-#define NEB_FOG_VISIBILITY_MULT_SHIELD			1.2f
-#define NEB_FOG_VISIBILITY_MULT_GLOWPOINT		1.2f
-#define NEB_FOG_VISIBILITY_MULT_BEAM(size)		4.0f + (size / 10)
-#define NEB_FOG_VISIBILITY_MULT_B_MUZZLE(size)  NEB_FOG_VISIBILITY_MULT_BEAM(size)
-#define NEB_FOG_VISIBILITY_MULT_PARTICLE(size)  1.0f + (size / 12)
-#define NEB_FOG_VISIBILITY_MULT_SHOCKWAVE		2.5f
-#define NEB_FOG_VISIBILITY_MULT_FIREBALL(size)	1.2f + (size / 12)
```

Comment:
Instead of having one value that combines the offset and divisor, I would rather have two values.  It makes things clearer and is also more flexible.

Reply:
You mean those `(size / 10)` values right? So you would have a constant factor and a separately controllable size factor?

Reply:
Right, I may have worded it badly.  Visibility A and visibility B, where the factor = A + (size / B)